README.md

```diff
@@ -1,8 +1,14 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
-
 ## Getting Started
 
-First, run the development server:
+First, populate environment variables
```

Comment:
Lets add a file `.env.default` that people can copy into .env (with the same content you show in the README